src/test/java/com/google/api/codegen/testdata/go_library.baseline

```diff
@@ -159,15 +127,15 @@ func NewClient(ctx context.Context, opts ...gax.ClientOption) (*Client, error) {
 
 // Close closes the connection to the API service. The user should invoke this when
 // the client is no longer required.
-func (c *Client) Close() error {
+func (c *LibraryClient) Close() error {
     return c.conn.Close()
 }
 
 // Path templates.
 
 // ShelfPath returns the path for the shelf resource.
-func ShelfPath(shelf string) string {
-    path, err := shelfPathTemplate.Render(map[string]string{
+func (c *LibraryClient) ShelfPath(shelf string) string {
```

Comment:
I've moved path template methods back under the client struct because they can conflict with other services. Each service defines a set of template strings which are available to the generator, which operates on a per-service basis. To prevent conflicts, the private vars are made unique by prefixing with the service name.

A possible solution to avoid this is to have a separate file that contains path template methods, which ignores equivalent template strings defined by multiple services (for example, 2 services could define a project template). I'm not sure about doing it that way though... on intuition I think it might be more confusing.